Resources/Prototypes/_NF/Recipes/Cooking/meal_recipes.yml

```diff
@@ -0,0 +1,1742 @@
+  name: double rat kebab recipe
+  result: FoodMeatRatdoubleKebab
+  time: 20
+  solids:
+    FoodMeatRat: 2
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeHumanKebab
+  name: human kebab recipe
+  result: FoodMeatHumanKebab
+  time: 15
+  solids:
+    TorsoHuman: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeLizardKebab
+  name: lizard kebab recipe
+  result: FoodMeatLizardtailKebab
+  time: 15
+  solids:
+    FoodMeatLizard: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeFoodMealTaco
+  name: taco recipe
+  result: FoodMealTaco
+  time: 10
+  solids:
+    FoodDoughSlice: 1
+    FoodTomato: 1
+    FoodOnionSlice: 2
+    FoodCheeseSlice: 1
+    FoodMeatCutlet: 1
+
+- type: microwaveMealRecipe
+  id: RecipeFoodBakedChevreChaud
+  name: chevre chaud recipe
+  result: FoodBakedChevreChaud
+  time: 5
+  solids:
+    FoodChevreSlice: 1
+    FoodBreadBaguetteSlice: 1
+
+- type: microwaveMealRecipe
+  id: RecipeCookedSpaceshroom
+  name: cooked spaceshroom recipe
+  result: FoodSpaceshroomCooked
+  time: 5
+  solids:
+    FoodSpaceshroom: 1
+
+- type: microwaveMealRecipe
+  id: RecipeCookedTortilla
+  name: cooked tortilla recipe
+  result: FoodMexicanTortilla
+  time: 5
+  solids:
+    FoodDoughCornmealSlice: 1
+
+- type: microwaveMealRecipe
+  id: RecipeBoringTaco
+  name: boring taco recipe
+  result: FoodMexicanBoringTaco
+  time: 10
+  solids:
+    FoodMexicanTortilla: 1
+    FoodMeatCutlet: 1
+    FoodCheeseSlice: 1
+
+- type: microwaveMealRecipe
+  id: RecipeTacoSupreme
+  name: taco supreme recipe
+  result: FoodMexicanTacoSupreme
+  time: 10
+  solids:
+    FoodMexicanTortilla: 1
+    FoodMeatCutlet: 1
+    FoodCheeseSlice: 1
+    FoodTomato: 1
+    FoodCabbage: 1
+
+- type: microwaveMealRecipe
+  id: RecipeBeanBurrito
+  name: bean burrito recipe
+  result: FoodMexicanBeanBurrito
+  time: 10
+  solids:
+    FoodMexicanTortilla: 1
+    FoodSoybeans: 2
+
+- type: microwaveMealRecipe
+  id: RecipeBeefurrito
+  name: beefurrito recipe
+  result: FoodMexicanBeefurrito
+  time: 10
+  solids:
+    FoodMexicanTortilla: 1
+    FoodMeatCutlet: 1
+    FoodSoybeans: 1
+    FoodCheeseSlice: 1
+
+- type: microwaveMealRecipe
+  id: RecipeCheesurrito
+  name: cheesurrito recipe
+  result: FoodMexicancheesurrito
+  time: 10
+  solids:
+    FoodMexicanTortilla: 1
+    FoodSoybeans: 1
+    FoodCheeseSlice: 2
+
+- type: microwaveMealRecipe
+  id: RecipeNachos
+  name: nacho recipe
+  result: FoodMexicanNachos
+  time: 10
+  solids:
+    FoodMexicanChip: 5
+    FoodCheeseSlice: 2
+
+- type: microwaveMealRecipe
+  id: RecipeSalsa
+  name: salsa recipe
+  result: FoodMexicansalsa
+  time: 10
+  solids:
+    FoodMexicanChip: 5
+    FoodTomato: 1
+    FoodOnionSlice: 1
+    FoodChili: 1
+
+- type: microwaveMealRecipe
+  id: RecipeElDiablo
+  name: el diablo burrito recipe
+  result: FoodMexicaneldiablo
+  time: 10
+  solids:
+    FoodMexicanTortilla: 1
+    FoodSoybeans: 1
+    FoodChili: 2
```

Comment:
this recipe is a large meal and has funny properties, should make it require some additional unique ingredient to craft